g3/generate/prompts/commit/creator.py

```diff
@@ -5,58 +5,85 @@
 from g3.generate.prompts.commit.examples.ruby import ruby_sample
 from g3.generate.prompts.commit.examples.node import node_sample
 from g3.generate.prompts.commit.examples.python import python_sample
+from g3.main import MessageTone
 
 
 class Creator:
 
-    def __init__(self, tone, jira, include):
-        self.tone = tone
-        self.jira = jira
-        self.include = include
+    def __init__(self):
         self.ruby_sample = ruby_sample
         self.node_sample = node_sample
         self.python_sample = python_sample
         openai.api_key = os.environ.get("OPENAI_API_KEY")
 
-    def create(self, code, stack: str = "python") -> str:
+    def create(self, code, tone: MessageTone, jira: str, include: str, stack: str = "python", temperature: float = 0,
+               model: str = 'gpt-4') -> str:
```

Comment:
We can skip temperature and model args by importing the [config](https://github.com/Workable/g3/blob/main/g3/main.py#L13) instance. `stack` is also not expected to be provided, so I would remove it too.